examples/MobileNetV2/MobileNetV2.cpp

```diff
@@ -63,37 +63,48 @@ const ml::Operand MobileNetV2::BuildConv(const ml::GraphBuilder& builder,
 
-    std::vector<int32_t> newShape = mLayout == "nchw" ? std::vector<int32_t>({1, -1, 1, 1})
-                                                      : std::vector<int32_t>({1, 1, 1, -1});
-    const ml::Operand reshapedBias = builder.Reshape(convBias, newShape.data(), newShape.size());
-
-    const ml::Conv2dOptions* conv2dOptions = options != nullptr ? options->AsPtr() : nullptr;
-    const ml::Operand conv = builder.Conv2d(input, convWeights, conv2dOptions);
-    const ml::Operand add = builder.Add(conv, reshapedBias);
-    if (fused) {
-        ml::ClampOptions clampOptions;
-        float min = 0;
-        auto minConstant = std::make_shared<std::vector<char>>(sizeof(float));
-        std::memcpy(minConstant->data(), &min, sizeof(float));
-        mConstants.push_back(minConstant);
-        float max = 6;
-        auto maxConstant = std::make_shared<std::vector<char>>(sizeof(float));
-        std::memcpy(maxConstant->data(), &max, sizeof(float));
-        mConstants.push_back(maxConstant);
-        clampOptions.minValue =
-            utils::BuildConstant(builder, {}, minConstant->data(), sizeof(float));
-        clampOptions.maxValue =
-            utils::BuildConstant(builder, {}, maxConstant->data(), sizeof(float));
-        const ml::Operand clamp = builder.Clamp(add, &clampOptions);
-        return clamp;
+    ml::ClampOptions clampOptions;
+    auto minConstant = SHARED_DATA_TYPE(new std::vector<char>(sizeof(float)));
+    *(reinterpret_cast<float*>(minConstant->data())) = 0;
+    mConstants.push_back(minConstant);
+    auto maxConstant = SHARED_DATA_TYPE(new std::vector<char>(sizeof(float)));
+    *(reinterpret_cast<float*>(maxConstant->data())) = 6;
+    mConstants.push_back(maxConstant);
+    clampOptions.minValue = utils::BuildConstant(builder, {}, minConstant->data(), sizeof(float));
+    clampOptions.maxValue = utils::BuildConstant(builder, {}, maxConstant->data(), sizeof(float));
+
+    if (!mFused) {
+        std::vector<int32_t> newShape = mLayout == "nchw" ? std::vector<int32_t>({1, -1, 1, 1})
+                                                          : std::vector<int32_t>({1, 1, 1, -1});
+        const ml::Operand reshapedBias =
+            builder.Reshape(convBias, newShape.data(), newShape.size());
+
+        const ml::Conv2dOptions* conv2dOptions = options != nullptr ? options->AsPtr() : nullptr;
+        const ml::Operand conv = builder.Conv2d(input, convWeights, conv2dOptions);
+        const ml::Operand add = builder.Add(conv, reshapedBias);
+        if (relu6) {
+            return builder.Clamp(add, &clampOptions);
+        } else {
+            return add;
+        }
     } else {
-        return add;
+        utils::Conv2dOptions fusedOptions;
+        if (options != nullptr) {
+            fusedOptions = *options;
+        }
+        fusedOptions.bias = convBias;
+        if (relu6) {
+            fusedOptions.activation = utils::CreateActivationOperator(
```

Comment:
Should you use `builder.ClampOperator` directly? It would align with L.86.

Reply:
OK, thanks! I will.